### PR TITLE
Info message update.

### PR DESCRIPTION
**Pull request by brianseek**
## Summary
Moves message out of info box to the top of the page.

## Related issue(s)
- department-of-veterans-affairs/va.gov-team#54279

## Screenshots

| travel pay without VAOS design | travel pay with VAOS design |
| --- | --- |
| localhost_3001_health-care_appointment-check-in__id=46bebc0a-b99c-464f-a5c5-560bc9eae287 (10) | localhost_3001_health-care_appointment-check-in_complete_0001-0001 (1) |
| | |

| no travel pay without VAOS design | no travel pay with VAOS design |
| --- | --- |
| localhost_3001_health-care_appointment-check-in_complete_0001-0001 (3) | localhost_3001_health-care_appointment-check-in_complete_0001-0001 (4) |
| | |

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature
- [ ]  [Accessibility foundational testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### CODE REVIEW

File: src/applications/check-in/day-of/pages/Confirmation/CheckInConfirmation.jsx [hunk 1]
```diff
@@ -160,20 +166,6 @@ const CheckInConfirmation = props => {
           )}
         </ol>
 
-        <va-alert
-          background-only
-          show-icon
-          data-testid="confirmation-alert"
```
Comment (wullaski) [inline src/applications/check-in/day-of/pages/Confirmation/CheckInConfirmation.jsx:166]: The test for this fails locally. I wonder why it didn't in the pipeline. We could probably just move the test id to the paragraph tag above.

**Review by wullaski — APPROVED** [review]
LGTM! Just need to make sure the cypress test passes again.

File: src/applications/check-in/day-of/pages/Confirmation/CheckInConfirmation.jsx (hunk 1 above)
Comment (brianseek) [reply src/applications/check-in/day-of/pages/Confirmation/CheckInConfirmation.jsx:166]: Great eye 👁️ !

I've updated the function and added back a testID

**Review by wullaski — APPROVED** [review]
LGTM! 🚢

**Comment by brianseek** [comment]
Closing, feature not changing for now.

**Pull request closed without merging**